net-lib/net/dns.rkt

```diff
@@ -14,6 +16,28 @@
            (->* ((or/c ip-address? ip-address-string?) string?)
                 (#:ipv6? any/c)
                 ip-address-string?)]
+          [dns-get-srv
+           (->* ((or/c ip-address? ip-address-string?)
+                 string?
+                 string?)
+                (string?)
+                (listof (struct/c srv-rr
+                                  (integer-in 0 65535)
+                                  (integer-in 0 65535)
+                                  (integer-in 0 65535)
+                                  string?)))]
+          [dns-lookup
```

Comment:
I've decided to not export it for now. We can revisit this in future if we like. It would be useful to expose for any time people want a different kind of RR not provided by the library.